README.md

```diff
@@ -20,25 +20,39 @@ file a [feature request][report-feature-request]!
 
 ![Screen capture of tool](art/screencap.gif)
 
+## Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute]](CONTRIBUTING.md).
+<a href="graphs/contributors"><img src="https://opencollective.com/shapeshifter/contributors.svg?width=890" /></a>
+
+
 ## Backers
 
 Support us with a monthly donation and help us continue our activities. 
 Funds will go towards hiring new developers to work on awesome features! 
 We'll display your logo below with a link to your site. 
 🙏 [[Become a backer](https://opencollective.com/shapeshifter#backer)]
 
-<a href="https://opencollective.com/shapeshifter/backer/0/website" target="_blank"><img src="https://opencollective.com/shapeshifter/backer/0/avatar.svg"></a>
-<a href="https://opencollective.com/shapeshifter/backer/1/website" target="_blank"><img src="https://opencollective.com/shapeshifter/backer/1/avatar.svg"></a>
-<a href="https://opencollective.com/shapeshifter/backer/2/website" target="_blank"><img src="https://opencollective.com/shapeshifter/backer/2/avatar.svg"></a>
-<a href="https://opencollective.com/shapeshifter/backer/3/website" target="_blank"><img src="https://opencollective.com/shapeshifter/backer/3/avatar.svg"></a>
-<a href="https://opencollective.com/shapeshifter/backer/4/website" target="_blank"><img src="https://opencollective.com/shapeshifter/backer/4/avatar.svg"></a>
-<a href="https://opencollective.com/shapeshifter/backer/6/website" target="_blank"><img src="https://opencollective.com/shapeshifter/backer/6/avatar.svg"></a>
+Thank you to all our backers! 🙏 [[Become a backer](https://opencollective.com/shapeshifter#backer)]
+
+<a href="https://opencollective.com/shapeshifter#backers" target="_blank"><img src="https://opencollective.com/shapeshifter/backers.svg?width=890"></a>
```

Comment:
This badge looks great, but is there one for sponsors too? Do we need to update the README each time there is a new sponsor?